src/legacy/core_plugins/kibana/server/tutorials/coredns_logs/index.js

```diff
@@ -46,7 +46,7 @@ export function corednsLogsSpecProvider(server, context) {
         learnMoreLink: '{config.docs.beats.filebeat}/filebeat-module-coredns.html',
       },
     }),
-    //TODO: euiIconType: 'logoCoredns',
+    euiIconType: '/plugins/kibana/home/tutorial_resources/logos/coredns.svg',
```

Comment:
I understand this is intended, isn't it?

Reply:
Yep, I found that the logo was added for metrics but was not being used for logs.